src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQNotifierConfig.java

```diff
@@ -163,7 +155,7 @@ public void setEnableNotifier(boolean enableNotifier) {
      *
      * @return true if verbose logging is enabled.
      */
-    public boolean isVerboseLoggingEnabled() {
+    public boolean getEnableVerboseLogging() {
```

Comment:
What is the reason for renaming these? Starting a method name with "is", is for me an easy way of grasping what the method is doing.

Reply:
I agree with you, although JCasC doesn't. It expects a getter and setter matching the property name, i.e. for "enableVerboseLogging"; getEnableVerboseLogging and setEnableVerboseLogging are expected for the configuration to work. Don't know if there is a workaround :/